web/content/docs/tools/workflows/create-a-simple-parallel-model/partmesh.pandoc

```diff
@@ -0,0 +1,17 @@
+
+# OpenGeosys  parallel simulation model preparation command tool:  partmesh
+
+**partmesh**  is a command tool to generate parted mesh from vtu files. It will run **mpmetis** to get a parted index for nodes, and then write parted meshes&nodes&elements&properties files as binary/ASCII format. Here introduce the options and workflow of this command tool:
+
+option | detail
+-------- | -----
+-i | define the input mesh file, most time is a vtu file
```

Comment:
While I think, that an overview of options is nice, this documentation will go out of sync with the `partmesh` tool as it changes. Better to improve the `partmesh` option descriptions (if needed) and not to repeat it externally. It will diverge.
Looking into the individual descriptions, they are just slightly changed from the `partmesh --help` output and do not add any new information.